web/src/date-utils.ts

```diff
@@ -54,31 +33,46 @@ export const dateToFormat = (date: Date | number, format: DateFormat): string =>
   if (isDateValid(date)) {
     switch (format) {
       case DateFormat.TimeShort:
-        return timeShortFormatter.format(date);
+        {
```

Comment:
Nit, some `{` are on the same line as the `case` statement, some are on their own line.

I guess they should be on the same line to mirror our `if` / `else` statements.

Ideally eslint would check this.